src/PageGenerator/project.json

```diff
@@ -5,18 +5,11 @@
     "url": "git://github.com/aspnet/diagnostics"
   },
   "dependencies": {
-    "Microsoft.AspNet.Diagnostics.Elm": "1.0.0-*",
-    "Microsoft.AspNet.Diagnostics.Entity": "7.0.0-*",
     "Microsoft.AspNet.Razor": "4.0.0-*"
   },
 
   "frameworks": {
-    "dnx451": { },
-    "dnxcore50": {
```

Comment:
It's a tool, but it needs to be able to run cross-plat, especially now that it runs as part of the build, and some platforms have only CoreCLR.

@pranavkm do you have more info on what was broken?